dice-mfg-msgs/src/lib.rs

```diff
@@ -223,6 +223,20 @@ impl MfgMessage {
     pub const MAX_ENCODED_SIZE: usize =
         corncobs::max_encoded_len(Self::MAX_SIZE);
 
+    pub fn as_str(&self) -> &str {
```

Comment:
Strum is handy for this: https://docs.rs/strum/latest/strum/derive.IntoStaticStr.html.  If you end up changing WrongMsg to have a MfgMessage argument instead of a String, you can derive Display instead since the driving use case for this method seems to be displaying as part of a format string.